dataprofiler/profilers/profile_builder.py

```diff
@@ -1112,6 +1114,19 @@ def _save_helper(self, filepath: str | None, data_dict: dict) -> None:
         # Restore all data labelers
         self._restore_data_labelers(data_labelers)
 
+    def _json_save_helper(self, filepath: str) -> None:
+        """
+        Save profiler to disk.
+
+        :param filepath: Path of file to save to
+        :type filepath: String
+        :return: None
+        """
+        if filepath is None:
+            filepath = ""
+        with open(filepath, "w") as f:
+            json.dump(self, f, cls=ProfileEncoder)
+
     def save(self, filepath: str = None) -> None:
```

Comment:
Here we need to add the type: str | None with default None where we can choose '"pickle "| "json"`